quiz/backend/install-poetry.py

```diff
@@ -39,7 +39,7 @@
 from functools import cmp_to_key
 from io import UnsupportedOperation
 from pathlib import Path
-from typing import Optional
+from typing import Optional, cast, Union, Iterator
```

Comment:
dlaczego dodałeś to do instalatora poetry?

Reply:
15 błędów wyrzygał mypy do tego pliku, te które się dało i nie maja zależności systemowych poprawiłem, a reszta poszła jako ignore 🙂

Reply:
a no to jednak jestem dzbanem, bo ja myślałem że to jest wasz customowy skrypt, a to jest installer z poetry 🥴 Więc, albo źle ustawiłem mypy, albo idę robić pr do poetry 🙃

Reply:
Dawaj pr do poetry 😀